osmaxx/conversion/converters/converter_gis/sql_tables.py

```diff
@@ -0,0 +1,36 @@
+VIEW_NAMES = [
```

Comment:
Also, why is the module called `sql_tables`, when there's nothing about SQL `TABLE`s in it?